test/test_utils.py

```diff
@@ -7,8 +7,9 @@
 import numpy as np
 import pytest
 import torch
-import torchvision.transforms.functional as F
 import torchvision.utils as utils
+import torchvision.transforms.functional as F
+from torchvision.transforms.v2.functional import to_dtype
```

Comment:
You'll need to re-order the imports, check out https://github.com/pytorch/vision/actions/runs/7214131414/job/19676695616?pr=8150, or run the pre-commit hooks locally (check our contributing instructions)